test/v1/validation/AgentDetailsValidatorSpec.scala

```diff
@@ -48,8 +48,7 @@ class AgentDetailsValidatorSpec extends BaseSpec {
 
     "passed false and Some name (with incorrect name length) should not succeed with 2 errors" in {
       val model = AgentDetailsModel(false, Some(""))
-      leftSideError(model.validate).errorMessage shouldBe
-        errorMessages(AgentNameLengthError("").errorMessage, AgentNameSuppliedError("").errorMessage)
+      leftSideError(model.validate, 1).errorMessage shouldBe AgentNameSuppliedError("").errorMessage
```

Comment:
See the comment for the country code test as we want to do the same here and retain the test that both errors are returned